pkg/envoy/ingress_translator.go

```diff
@@ -147,6 +149,28 @@ func classFilter(ingresses []v1beta1.Ingress, ingressClass []string) []v1beta1.I
 	return is
 }
 
+func validIngressFilter(ingresses []v1beta1.Ingress) []v1beta1.Ingress {
+	vi := make([]v1beta1.Ingress, 0)
+
+	for _, i := range ingresses {
+		ing := make([]v1.LoadBalancerIngress, 0)
```

Comment:
something else we should be checking for here is ingress.Spec.Rules, if there are no rules with a host then we can't use this ingress